openbb_terminal/forex/forex_controller.py

```diff
@@ -229,7 +229,10 @@ def call_candle(self, other_args: List[str]):
             "--ma",
             dest="mov_avg",
             type=str,
-            help=translate("stocks/CANDLE_mov_avg"),
+            help=translate(
```

Comment:
I've kept the `translate` here, although I'm not sure how this works.

Reply:
I think it translates from english in `en.yml` file inside `i18n` folder